src/example_report.rs

```diff
@@ -0,0 +1,90 @@
+use std::convert::From;
+
+#[cfg(feature = "use_expectest")]
+use expectest::core::TestResult as ExpectestResult;
+
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub struct Failure {
+    pub message: Option<String>,
+}
+
+impl Failure {
+    pub fn new<S>(message: Option<S>) -> Self
+        where String: From<S>
```

Comment:
Sure. To be honest I didn't have `ToString` on my screen at all, when I wrote this. 😇